packages/editor-ui/src/views/NodeView.vue

```diff
@@ -2737,6 +2792,9 @@ export default mixins(
 				try {
 					this.initNodeView();
 					await this.initView();
+					if (window.top) {
+						window.top.postMessage(JSON.stringify({command: 'n8nReady'}), '*');
```

Comment:
can you add n8n version here... seems to me like this might be useful to future proof this